src/components/Checkbox.tsx

```diff
@@ -52,25 +60,37 @@ type ControlledCheckboxProps = Omit<CheckboxProps, 'value' | 'onChangeValue'> &
   name: string
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   control: Control<any>
+  errors?: FieldErrors
   isRequired?: boolean
 }
 
 export const ControlledCheckbox = ({
   name,
   control,
+  errors,
   isRequired = false,
   ...props
 }: ControlledCheckboxProps) => {
+  const error = !!get(errors, name)
+  console.log(error)
+
   const handlePress = useCallback(
     (field: ControllerRenderProps) => (newValue: boolean) => field.onChange(newValue),
     []
   )
 
   const renderCheckbox = useCallback(
     ({ field }: ControllerFieldProps) => {
-      return <Checkbox {...props} onChangeValue={handlePress(field)} value={field?.value} />
+      return (
+        <Checkbox
```

Comment:
Shouldn't we also render error message under the checkbox? For example, "you need to accept terms and conditions to create account" 🤔

Reply:
As we discussed I changed it to make the text red when an error appears.